src/stats/basic_stats.rs

```diff
@@ -0,0 +1,141 @@
+use ndarray::*;
+use std;
+
+/// Retrieves the min element from an ndarray Array
+/// 
+/// # Examples
+/// ```
+/// # #[macro_use]
+/// # extern crate ndarray;
+/// # extern crate num_ru;
+/// # extern crate chrono;
+/// use ndarray::*;
+/// use num_ru::stats::basic_stats::*;
+/// use chrono::{NaiveDate, NaiveDateTime};
+///
+/// # fn main(){
+/// let arr = array![[[5, 6], [7, 0]], [[1, 2], [3, 4]]];
+/// assert_eq!(amin(&arr), 0);
+/// let dt1: NaiveDateTime = NaiveDate::from_ymd(2016, 7, 8).and_hms(9, 10, 11);
+/// let dt2: NaiveDateTime = NaiveDate::from_ymd(2018, 7, 8).and_hms(9, 10, 11);
+/// let dt3: NaiveDateTime = NaiveDate::from_ymd(2016, 7, 8).and_hms(13, 10, 11);
+/// let arr2 = array![dt3, dt1, dt2];
+/// assert_eq!(amin(&arr2), dt1);
+/// # }
+/// ```
+pub fn amin<A, D>(arr: &Array<A, D>) -> A
```

Comment:
Have you thought about how to parallelize functions such as this one? That could be important for performance.